src/App.tsx

```diff
@@ -17,18 +46,20 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter query={query} filterBy={filterBy} setFilterBy={setFilterBy} setQuery={setQuery} applyQuery={applyQuery} />
```

Comment:
move each props to a new line, if you have three or more props each of them should be on the new line, fix in all places